HOWTO.md

```diff
@@ -58,23 +58,23 @@ exports.handler = (event, context, callback) => {
         case "GET":
             
             if(id) {
-                callback(null, "This is a READ operation on product ID " + id);
+                callback(null, { body: "This is a READ operation on product ID " + id});
```

Comment:
Awesome! It is generally a good practice to manually return statusCode = 200 for successful calls. Can you make that change as well?